quarkus/config-api/src/main/java/org/keycloak/config/TruststoreOptions.java

```diff
@@ -14,7 +14,7 @@ public class TruststoreOptions {
     public static final Option<HostnameVerificationPolicy> HOSTNAME_VERIFICATION_POLICY = new OptionBuilder<>("tls-hostname-verifier", HostnameVerificationPolicy.class)
```

Comment:
We could also leverage the values deprecation logic that was added today in #27367.

Reply:
Done